### DISCUSSION TITLE
Why does Cyberduck take ~5-10 seconds to start on Windows?

### DISCUSSION THREAD

**Discussion by alturic** [iterate-ch/cyberduck#15707] [Q&A]
I believe Cyberduck is ported from Mac(?) but I'm unsure if this would be the cause of this but ever since I started using Cyberduck a couple of years ago it has always taken a solid 5-10 seconds before starting and showing my ~15 server bookmarks.

I suppose I'm just curious after all this time why exactly it takes so long to load?

**Comment by tatze96** [comment]
We are booting up a complete Java VM in addition to the regular startup of .NET Framework and loading several hundred MBs of dependency DLLs.

duplicates #12630
